createas.c

```diff
@@ -1203,7 +1203,7 @@ CreateIndexOnIMMV(Query *query, Relation matviewRel, bool is_create)
 					(errmsg("could not create an index on immv \"%s\" automatically",
 							RelationGetRelationName(matviewRel)),
 					 errdetail("This target list does not have all the primary key columns, "
-							   "or this view does not contain GROUP BY or DISTINCT clause."),
+							   "or this view does not contain DISTINCT clause."),
```

Comment:
Could you please explain why is this change required?

Reply:
I think that currently GROUP BY is not icluded in this message.  
But it needs discussion, so I will create new  pull request.